corehq/apps/app_manager/models.py

```diff
@@ -4217,7 +4217,7 @@ def form_record_url(self):
         return '/a/%s/api/custom/pact_formdata/v1/' % self.domain
 
     @absolute_url_property
-    def profile_url(self):
+    def hq_profile_url(self):
```

Comment:
given that it was a source of confusion in https://github.com/dimagi/commcare-hq/pull/9554/commits/34eb33cf1ecb854fe58187045211fa7e5b7d8a8c maybe add a comment and link to the bug?